writer.go

```diff
@@ -59,6 +59,11 @@ func (w *Writer) Write(b []byte) (int, error) {
 	return w.writeAndRetry(w.priority, string(b))
 }
 
+// RawWrite sends a log message with a custom priority
+func (w *Writer) RawWrite(p Priority, b []byte) (int, error) {
```

Comment:
Regarding test coverage, Write method is not covered either. Could you suggest what should I test in this case?

I am not that good at go, so I have trouble expressing my tests. At most what I could think of testing is "does Write produce a log message with default priority" and "does <my new write> produce a log message with given priority". Yet I could not understand existing test cases to implement my tests using code from there. Alternatively to my first question, you could simply help me to understand existing cases.